my-app/src/components/TasksList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { deleteTaskAction} from '../redux'
+
+
+const TasksList = () => {
+    const tasks = useSelector((state) => state.tasks)
+    const dispatch = useDispatch()
+    const deleteTask = (id) => dispatch(deleteTaskAction(id))   
```

Comment:
convert to be a closure: `(id) => () => dispatch(deleteTaskAction(id))` and wrap with `useCallback` so it gets memoized